.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
Well, the issue with this setup would be versionning for changes in Mink. But yeah, this is something I already thought about.